pages/other-terms.mdx

```diff
@@ -60,7 +60,7 @@ One of three properties of [zero-knowledge proofs](#zero-knowledge-proof), compl
 A sharding design proposed for Ethereum. Instead of providing more space for transactions, Ethereum sharding provides more space for [blobs](#blobs) of data. Verifying a blob simply requires checking that the blob is available - that it can be downloaded from the network. The data space in these blobs is expected to be used by layer-2 rollup protocols.
 In comparison to other sharding mechanisms, where there are a fixed number of shards with distinct blocks and distinct block proposers, in Danksharding there is only one proposer that chooses all transactions and all data that go into that slot.
 
-## Deterministic
+## Determinism
```

Comment:
Hmm.. I feel like "deterministic" describes this better? or do you disagree.

Reply:
My thinking was that "determinism" is better because typically glossaries are made up of nouns rather than adjectives unless it's a very specific term. We have "trustlessness", "completeness" etc in the glossary, I thought we could stick to having terms as nouns.

Obviously it's not a big deal, we can leave it as it is.